File: reservation-service/src/main/java/rs/raf/reservationservice/domain/Firm.java
package rs.raf.reservationservice.domain;

import javax.persistence.*;
import lombok.*;

@Entity
@Table(name = "firm")
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class Firm {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    @Column(name = "firm_name")
    private String firm_name;
    @Column(name = "description")
    private String description;
    @Column(name = "number_of_vehicles")
    private Integer number_of_vehicles;
    @Column(name = "city")
    private String city;
}

File: ReservationServiceClient/model/reservationservice/base/Firm.cs

public class Firm {
    public long id{ get; set; }
    public string firm_name{ get; set; }
    public string description{ get; set; }
    public int number_of_vehicles{ get; set; }
    public string city { get; set; }
    public Firm(long id, string firm_name, string description, int number_of_vehicles, string city) {
        this.id = id;
        this.firm_name = firm_name;
        this.description = description;
        this.number_of_vehicles = number_of_vehicles;
        this.city = city;
    }
    public Firm() {

    }
}
